source/Engine/ResourceTypes/SoundFormats/WAV.cpp

```diff
@@ -5,34 +5,24 @@
 #include <Engine/Diagnostics/Memory.h>
 #include <Engine/IO/ResourceStream.h>
 
-struct RIFF_Header {
-    Uint32        Magic;
-    Uint32        ChunkSize;
-    Uint32        Format;
-};
-struct fmt_Header {
-    Uint32        Magic;
-    Uint32        Size;
-    Uint32        AudioFormat;
-};
 struct WAVheader {
     /* RIFF Chunk Descriptor */
-    Uint8         RIFF[4];        // RIFF Header Magic header
-    Uint32        ChunkSize;      // RIFF Chunk Size
-    Uint8         WAVE[4];        // WAVE Header
+    // Uint8         RIFF[4];        // RIFF Header Magic header
+    // Uint32        ChunkSize;      // RIFF Chunk Size
+    // Uint8         WAVE[4];        // WAVE Header
```

Comment:
If those fields are no longer read then delete those lines

Reply:
Done as a fixup so it's easier to review, can rebase and squash if you want me to.

Reply:
Fine by me.